frontend/src/views/AboutView.vue

```diff
@@ -1,65 +1,129 @@
-  <v-container>
-    <v-row align="center" justify="center">
-      <v-col v-for="member in members" :key="member.name" cols="auto">
-        <v-card>
-          <v-img :src="`${member.image}`" :width="400" aspect-ratio="1" cover
-            lazy-src="https://www.hydroshare.org/static/static/img/home-page/carousel/bg3.jpg">
-            <template v-slot:placeholder>
-              <div class="d-flex align-center justify-center fill-height">
-                <v-progress-circular color="grey-lighten-4" indeterminate></v-progress-circular>
-              </div>
-            </template>
-          </v-img>
-          <v-card-title class="text-h6">
-            {{ member.name }}
-          </v-card-title>
-          <v-card-subtitle>
-            {{ member.position }}
-            <v-divider></v-divider>
-            {{ member.org }}
-          </v-card-subtitle>
-        </v-card>
-      </v-col>
-    </v-row>
+<v-container fluid>
+    <div>
+      <div class="mb-3">
+        <h3>Hydrologic Process Knowledge Platform</h3>
+        <p>
+          This project enables you to view and search hydrologic process
+          knowledge derived from perceptual models of research watersheds around
+          the globe.
+        </p>
+      </div>
+
+      <div class="mb-3">
+        <h3>Hydrologic Processes</h3>
+        <p>
+          Hydrologic processes describe the flow paths and mechanisms that move
+          water from the canopy and land surface to its release into streams,
+          deep groundwater or evapotranspiration. Hydrologists often communicate
+          process knowledge using a perceptual model, a qualitative summary of
+          our knowledge about the hydrologic system presented as a schematic
+          figure or a text description. Perceptual models are one of the best
+          tools that the hydrologic community has to organize hydrologic
+          knowledge, test hypotheses about runoff generation, design accurate
+          computer models to predict river flows, and identify knowledge gaps. 
+        </p>
+      </div>
+
+      <div class="mb-3">
+        <h3>Funding</h3>
+        <p>
+          This project is funded by the NSF Division of Earth Sciences Award
+          #2322510:
+          <div>
+            <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2322510"
+            >Synthesizing hydrologic process knowledge to determine global
+            drivers of dominant processes.</a>
+          </div>          
+        </p>
+      </div>
+
+      <div class="mb-3">
+        <h3>Project Team</h3>
+        <v-container fluid>
+          <v-row dense>
+            <v-col v-for="member in members" :key="member.name">
+              <v-card>
+                <v-img
+                  :src="member.img"
+                  class="align-end"
+                  gradient="to bottom, rgba(0,0,0,.1), rgba(0,0,0,.5)"
+                  height="350px"
+                  cover
+                >
```

Comment:
![image](https://github.com/user-attachments/assets/e3c60d82-358f-4584-925c-57d00c655140)
watch your use of [cover](https://vuetifyjs.com/en/components/images/#cover)